<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>39</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="5">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Engine 4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Engine 1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Engine 3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Engine 2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QSlider" name="sl_m4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSlider" name="sl_m2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QSlider" name="sl_m3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="sl_m1">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lbl_conn_to">
      <property name="text">
       <string>Connected to:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_curr_ip_port">
      <property name="text">
       <string>IP:Port</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>139</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="lbl_return_to_mm">
      <property name="text">
       <string>To Menu:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_return_to_mm">
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
